### PR TITLE
[Test-utils] Test String Utils

### PR DESCRIPTION
**Pull request by c-harding**

### CODE REVIEW

**Comment by c-harding** [comment]
Not tested mkString as it has been deleted in #22

**Review by jaylees14 — APPROVED** [review]

File: src/test/scala/temple/utils/StringUtilsTest.scala [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+class StringUtilsTest extends FlatSpec with Matchers {
+  "indent" should "add spaces to an empty string" in {
+    indent("") shouldEqual "  "
+  }
+  "indent" should "add n spaces to an empty string" in {
+    indent("", 1) shouldEqual " "
+    indent("", 3) shouldEqual "   "
+  }
+  "indent" should "add spaces to a single line" in {
+    indent("abcd") shouldEqual "  abcd"
```
Comment (willjvsmith) [inline src/test/scala/temple/utils/StringUtilsTest.scala:15]: tests will need to be updated if we ever change the default indent size, but I guess we'll never change it

**Review by willjvsmith — APPROVED** [review]
LGTM 🚀

**Pull request merged**